browser/brave_stats_updater_util.cc

```diff
@@ -82,8 +82,7 @@ base::Time GetYMDAsDate(base::StringPiece ymd) {
   const auto pieces = base::SplitStringPiece(ymd, "-", base::TRIM_WHITESPACE,
                                              base::SPLIT_WANT_NONEMPTY);
   DCHECK_EQ(pieces.size(), 3ull);
-  base::Time::Exploded time;
-  base::Time().LocalExplode(&time);
+  base::Time::Exploded time = {};
```

Comment:
Do we care that `base::Time::FromLocalExploded` below will adjust this?

Reply:
yeah, this is intentional - we just want to "decrypt" YMD which was also computed from local time (above in the same file)